scripts/kind-e2e/e2e.sh

```diff
@@ -19,13 +19,14 @@ function kind_clusters() {
             echo Creating cluster${i}, logging to ${logs[$i]}...
             (
             if [[ -n ${version} ]]; then
-                kind create cluster --image=kindest/node:v${version} --name=cluster${i} --wait=5m --config=${PRJ_ROOT}/scripts/kind-e2e/cluster${i}-config.yaml
+                kind create cluster --image=kindest/node:v${version} --name=cluster${i} --config=${PRJ_ROOT}/scripts/kind-e2e/cluster${i}-config.yaml
```

Comment:
Why do we remove the wait=5m? :)

Reply:
The control plain comes up without cni, --wait waits for control plain to be up and its never up until we apply weave cni.

Reply:
ohh, makes sense, thanks :-D